### PR TITLE
Pretty Crash Report

### PR DESCRIPTION
**Pull request by Dariensg**
-Made custom crash message for blacklist/whitelist crash

### CODE REVIEW

File: src/main/java/com/nekomaster1000/infernalexp/mixin/common/MixinCrashReport.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    private StackTraceElement[] stacktrace;
+
+    @Shadow
+    @Final
+    private Throwable cause;
+
+    @Inject(method = "getSectionsInStringBuilder", at = @At(value = "INVOKE", target = "Ljava/lang/StringBuilder;append(Ljava/lang/String;)Ljava/lang/StringBuilder;", ordinal = 0))
+    private void IE_checkForBiomeConfigCrash(StringBuilder builder, CallbackInfo ci) {
+        if (this.cause.getMessage().equals("/ by zero") && Arrays.stream(this.stacktrace).anyMatch(element -> element.getClassName().equals(ModNetherBiomeCollector.class.getName()) &&
+            element.getMethodName().equals("getRandomNetherBiomes"))) {
```
Comment (SwanX1) [inline src/main/java/com/nekomaster1000/infernalexp/mixin/common/MixinCrashReport.java]:
```suggestion
       if (
            this.cause.getMessage().equals("/ by zero") &&
            Arrays.stream(this.stacktrace).anyMatch(element -> element.getClassName().equals(ModNetherBiomeCollector.class.getName()) &&
            element.getMethodName().equals("getRandomNetherBiomes"))
        ) {
```
this makes it more readable, imo

**Review by SwanX1 — CHANGES_REQUESTED** [review]
i applaud you for putting the mixin in alphabetically, most people forget to do so

**Comment by SwanX1** [comment]
Also, want to mention that the build failed, but it was due to a maven repo fail, not your fault.

**Comment by Dariensg** [comment]
> 
> 
> i applaud you for putting the mixin in alphabetically, most people forget to do so

Lol I remembered you yelling at me for not doing it just the other day.

**Review by SwanX1 — APPROVED** [review]
LGTM

**Pull request merged**